### DISCUSSION TITLE
Confusing training process

### DISCUSSION THREAD

**Discussion by Pipe-Runner** [community/community#39544] [GitHub Education]
I got selected for the campus expert program and attended one of the onboarding calls. I probably missed some details, but I am not supposed to go through the training program, and I am not sure if I am supposed to wait for some follow-up email. 
Can someone point me in the right direction? Sorry if my questions make me sound ignorant.

**Comment by jfullstackdev** [comment]
I don't know if Support can help you

https://support.github.com/contact/account
